### PR TITLE
Fixed wrong audio file of a particular drum

### PR DESCRIPTION
**Pull request by Saatvik21**
## Related Issue
* The audio files were mismatched.
 
 Closes: #250 
 
 ### Describe the changes you've made
 I have fixed the above particular issue by changing the "kick" drum audio file to it's corresponding file.
 
 ### Describe if there is any unusual behaviour of your code(Write `NA` if there isn't)
 `NA`
 
 ### Checklist:
 * [x]  My code follows the style guidelines of this project.
 * [x]  I have performed a self-review of my own code.
 * [ ]  I have commented my code, particularly in hard-to-understand areas.
 * [ ]  I have made corresponding changes to the documentation.
 * [x]  New and existing unit tests pass locally with my changes.
 
 ### Screenshots
 
Screenshot_1

### CODE REVIEW

**Review by urvashi-code1255 — APPROVED** [review]
@Saatvik21 Its looking good to me :)

**Comment by Saatvik21** [comment]
> @Saatvik21 Please pull the recent changes. Your PR don't contain the recent changes.

Can you please tell me how to do that, actually I'm new to git that's why

**Comment by urvashi-code1255** [comment]
> Can you please tell me how to do that, actually I'm new to git that's why

`git pull upstream main`

**Comment by Saatvik21** [comment]
> > Can you please tell me how to do that, actually I'm new to git that's why
> 
> `git pull upstream main`

I just did this on my local branch ie "250" , is it done? or do I have to do it on my main branch of the forked repo?

**Comment by Saatvik21** [comment]
@urvashi-code1255 it's been 2 days and yet no response. Please review this so that I can make further issues.

**Comment by urvashi-code1255** [comment]
@Saatvik21 Please again pull the changes!

**Comment by Saatvik21** [comment]
> @Saatvik21 Please again pull the changes!

@urvashi-code1255 Done!

**Comment by Saatvik21** [comment]
@urvashi-code1255 @Vishal-raj-1 

I pulled the recent changes again. 

Please look into this.

**Comment by Saatvik21** [comment]
@ridsuteri @harshgupta20 @Vishal-raj-1 

Sir please look into this PR I made 7 days ago. It's not yet merged.

File: script.js [hunk 1]
```diff
@@ -215,11 +215,25 @@ let projectData = [
     projectImage: "assets/Images/IssueTracker.png",
     projectUrl: "public/IssueTracker.html",
   }
+<<<<<<< HEAD
```
Comment (urvashi-code1255) [inline script.js:218]: remove this

File: script.js [hunk 2]
```diff
@@ -215,11 +215,25 @@ let projectData = [
...
   {
     projectName: "Text to Voice Generator",
     projectImage: "assets/GIFs/textToVoice",
     projectUrl: "public/textToVoice.html",
   }
+=======
```
Comment (urvashi-code1255) [inline script.js:224]: remove this

File: script.js [hunk 3]
```diff
@@ -215,11 +215,25 @@ let projectData = [
...
+    projectName: "Pixels Game",
+    projectImage: "assets/GIFs/pixels.gif",
+    projectUrl: "public/pixels-game.html",
+  },
+ {
+        projectName: "love Calculator",
+        projectImage: "assets/GIFs/loveCalculator.gif",
+        projectUrl: "public/loveCalculator.html",
+    }
+>>>>>>> a3be3e50d72bf887301f257285f76a1ace405b4a
```
Comment (urvashi-code1255) [inline script.js:236]: remove this

**Review by urvashi-code1255 — COMMENTED** [review]
@Saatvik21 do the changes and preview the project locally that all the projects are visible or not

**Comment by Saatvik21** [comment]
> @Saatvik21 do the changes and preview the project locally that all the projects are visible or not

@urvashi-code1255 Done! All the projects are visible on my local machine.

**Review by ridsuteri — APPROVED** [review]

**Comment by ridsuteri** [comment]
thanks for contributing @Saatvik21 :)

**Comment by Saatvik21** [comment]
> thanks for contributing @Saatvik21 :)

@ridsuteri Thank you sir , but can you please add gssoc and level label for this PR?

**Comment by ridsuteri** [comment]
done :)

**Pull request merged**